Makefile

```diff
@@ -146,6 +146,8 @@ create-test-customer-no-providers: run-migrations
 	$(PYTHON) $(TOPDIR)/scripts/create_test_customer.py --no-providers --bypass-api || echo "WARNING: create_test_customer failed unexpectedly!"
 	kill -HUP $$(ps -eo pid,command | grep "manage.py runserver" | grep -v grep | awk '{print $$1}')
 
+load-test-customer-data:
```

Comment:
You may want to add this target to the `make help` target up above. Also, I was wondering if it would be possible to set a default for if the start date is not passed in to use just `today`? Like in the nise example here: https://github.com/project-koku/nise/blob/490cdbcb1bfd7aa3316e3c3e99b013d59b09a98a/tests/azure_static_report.yml#L4

If the default is a lot of work I would say skip it, but if its easy it could be a nice feature to just get this current month view in there quickly. Also, I have found its a lot easier to set defaults in bash than it is in makefile syntax.